apps/web/src/components/editor/manager.ts

```diff
@@ -94,6 +94,8 @@ const [useEditorManager] = createStore<EditorManager>(
 
 export { useEditorManager };
 
+export const editorManager = useEditorManager.getState();
```

Comment:
Bad idea. `useEditorManager.getState()` will only give you the current snapshot of the state. Just use `useEditorManager.getState` directly.

Reply:
When I use `useEditorManager.getState` directly, I was getting invalid hook call errors. Also, during my testing, using `editorManager` as I've done worked correctly. Let me explore more on this...

Reply:
Not sure how you were using it because that error has nothing to do with `useEditorManager.getState`.

Reply:
My bad, I was calling it like `useEditorManager()`. Fixed.